README.md

```diff
@@ -1,113 +1,59 @@
-Exposes APIs via AWS SDK for Java 1.x. Use alongside with `misk-aws-dynamodb`.
-
-Installing `InProcessDynamoDbModule` runs a DynamoDb Local instance in memory for your
-tests to run against. This module is recommended over `DockerDynamoDbModule` because there is less
-overhead in test execution performance.
-
-Installing `DockerDynamoDbModule` runs a DynamoDB Local instance in Docker for your tests to execute
-against.
-
-
-### misk-aws2-dynamodb
-
-Integrate with AWS DynamoDb using AWS SDK for Java 2.x. It should be safe to install side-by-side
-with `misk-aws-dynamodb` if you need to use features in both.
-
-Please read
-the [AWS SDK for Java 2.x Migration Guide](https://docs.aws.amazon.com/sdk-for-java/latest/migration-guide/what-is-java-migration.html)
-for more details.
-
-
-### misk-aws2-dynamodb-testing
-
-Integrate with this package to write tests for code that interacts with DynamoDb.
-Exposes APIs via AWS SDK for Java 2.x. Use alongside with `misk-aws2-dynamodb`.
-
-Installing `InProcessDynamoDbModule` runs a DynamoDb Local instance in memory for your
-tests to run against. This module is recommended over `DockerDynamoDbModule` because there is less
-overhead in test execution performance.
-
-Installing `DockerDynamoDbModule` runs a DynamoDB Local instance in Docker for your tests to execute
-against.
-
-
-### misk-service
+* Related Projects
+  * [misk-web][miskweb]
+  * [wisp][wisp]: wisp now is a part of Misk
  
-Bind Guava services with inter-service dependencies.
- 
-Any service can depend on any other service. ServiceManager won't start a service until the
-services it depends on are running.
-
-
-### misk-inject
- 
-Integrates Guice with Kotlin.
-
-Extending `KAbstractModule` instead of Guice's `AbstractModule` lets you use `KClass` instead
-of `java.lang.Class` and other conveniences.
-
-
-### misk-feature
- 
-Runtime feature flags. `misk-launchdarkly` is the reference implementation.
-
-
-### misk-jobqueue
- 
-A job queue with a high quality fake. `AwsSqsJobQueueModule` from `misk-aws` is the reference 
-implementation.
-
-
-### misk-events
- 
-An event publisher + consumer. There is no open source reference implementation at this time.
-
-## wisp-*
 
-The wisp-* modules contain no Dependency Injection based code (i.e. no Guice, etc) and back many misk-* module implementations.  
+# Overview
+## What is Misk
+Misk (Microservice Container in Kotlin) is an open source microservice container from Cash App.
+It allows you to quickly create a microservice in Kotlin or Java, and provides libraries for common
+concerns like serving endpoints, caching, queueing, persistence, distributed leasing and clustering.
+It also includes [the Wisp library](./wisp/README.md), which is a collection of Kotlin modules
+providing various features and utilities, including config, logging, feature flags and more.
+
+It has been powering hundreds microservices within Cash App since 2018.
```

Comment:
I'm not sure if this is something we would like to make public ?
Should we leave numbers out, and keep it more generic maybe? 

`Misk has been the backbone of Cash App's microservice infrastructure since 2018`

Reply:
I think this is fine to leave in, number of services doesn't leak anything critical.

Reply:
hmm, I haven't thought about this. My intention is to show it is a battle-proved framework, so people can have more confidence on it.
**hundreds of** [means a large number](https://dictionary.cambridge.org/dictionary/english/hundred-hundreds-of), so I guess it is safe to say 😆